svgator-php/autoload.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ *
+ */
+spl_autoload_register('SVGator_autoloader');
+
+function SVGator_autoloader($className) {
+	$className = str_replace('\\', DIRECTORY_SEPARATOR, $className);
+	if(strpos($className, 'SVGator') >= 0) {
```

Comment:
Let's check equality.. User mai also use SVGator as classname somewhere in his code